ext/native/thin3d/GLQueueRunner.cpp

```diff
@@ -947,9 +947,10 @@ void GLQueueRunner::PerformBindFramebufferAsRenderTarget(const GLRStep &pass) {
 void GLQueueRunner::CopyReadbackBuffer(int width, int height, Draw::DataFormat srcFormat, Draw::DataFormat destFormat, int pixelStride, uint8_t *pixels) {
 	// TODO: Maybe move data format conversion here, and always read back 8888. Drivers
 	// don't usually provide very optimized conversion implementations, though some do.
+	// Just need to be careful about dithering, which may break Danganronpa.
```

Comment:
I'd think reading back 8888 would be more likely to avoid any dithering issues...

Reply:
Should, as long as true color is true.  We do have this:

```c++
	if (PSP_CoreParameter().compat.flags().Force04154000Download && vfb->fb_address == 0x00154000) {
		trueColor = true;
	}
```

So it should be fine, but just adding the note so we're careful.  There was an issue with it dithering when it converted between depths.

-[Unknown]